test/integration/test_raw_api.py

```diff
@@ -538,6 +538,47 @@ def raw_api_test_helper(raw_api, should_cleanup_old_buckets):
             is_file_lock_enabled=False,
         )
 
+    # b2_100_continue
```

Comment:
this test will be replaced by whatever is merged as part of https://github.com/reef-technologies/b2-sdk-python/pull/300/files